python/ray/_private/ray_constants.py

```diff
@@ -437,3 +437,5 @@ def gcs_actor_scheduling_enabled():
 
 WORKER_SETUP_HOOK_ENV_VAR = "__RAY_WORKER_SETUP_HOOK_ENV_VAR"
 RAY_WORKER_SETUP_HOOK_LOAD_TIMEOUT_ENV_VAR = "RAY_WORKER_SETUP_HOOK_LOAD_TIMEOUT"
+
+RAY_DEFAULT_NODE_LABELS_KEYS = ["node_id", "node_ip"]
```

Comment:
Let's come up with a naming convention so that those system labels won't conflict with user labels.

How about all system labels have prefix `RAY_` (we don't allow users to create labels with this prefix).